.github/workflows/make-release-commit.yml

```diff
@@ -20,18 +17,17 @@ on:
           - preview
           - stable
       draft-release:
-        description: 'When creating release, should it be a draft?'
+        description: 'Create a draft release on GitHub'
```

Comment:
Wouldn't we want this to be true since we should generally give the release notes a manual inspection before fully publishing the release?

Reply:
I don't think there's any reason we can't edit after the fact. There's already a 20 minute delay between publishing the release and the artifacts being available. I feel like we'd often rather get that process started and do the editing while the build happens.